src/System.ServiceModel.Http/tests/ServiceModel/BasicHttpBindingTest.cs

```diff
@@ -35,6 +35,17 @@ public static void Default_Ctor_Initializes_Properties()
         Assert.True(TestHelpers.XmlDictionaryReaderQuotasAreEqual(binding.ReaderQuotas, new XmlDictionaryReaderQuotas()), "XmlDictionaryReaderQuotas");
     }
 
+    // BasicHttpSecurityMode.Message is not supported
+    [Fact]
+    public static void Create_HttpBinding_SecurityMode_Message()
+    {
```

Comment:
Nit: test method name should say what it does, like Create_BasicHttpBinding_SecurityMode_Message_Throws_PNSE or something like that.